src/plugins/intel_gna/src/common/graph_utils.hpp

```diff
@@ -87,7 +87,7 @@ inline bool is_aligned_split(const std::shared_ptr<ngraph::Node> input_op, size_
             offset += outputSize * limitations::Limitations::kBytesPerSplitElement;
         }
     }
-    return (offset == ALIGN64(offset));
+    return (offset == ALIGN(offset, limitations::Limitations::get_instance()->get_memory_alignment()));
```

Comment:
I see that in few places there is the same kind of invocation 
`offest*` == `ALIGN(offset, limitations::Limitations::get_instance()->get_memory_alignment()`

maybe it would be good to introduce method in limitations

e.g.
```
bool is_aligned(size_t offset){
      return (offset == ALIGN(get_memory_alignment()));
}
```
and use

`return limitations::Limitations::get_instance()->is_aligned(offset);`

Reply:
ok